packages/@aws-cdk/aws-batch/test/job-definition.test.ts

```diff
@@ -0,0 +1,118 @@
+import '@aws-cdk/assert/jest';
+import { ResourcePart } from '@aws-cdk/assert/lib/assertions/have-resource';
+import { InstanceClass, InstanceSize, InstanceType } from '@aws-cdk/aws-ec2';
+import { Repository } from '@aws-cdk/aws-ecr';
+import { EcrImage, LinuxParameters, MountPoint, Ulimit, Volume } from '@aws-cdk/aws-ecs';
```

Comment:
Please, don't import any classes (other than from the `@aws-cdk/core` package) directly. Always refer to them with a prefix.

Reply:
Got it 👍